examples/templates/preview_draft_template.py

```diff
@@ -0,0 +1,9 @@
+from sparkpost import SparkPost
+
+sp = SparkPost('YOUR API KEY')
+sub_data = {
+    'first_name': 'John',
+    'last_name': 'Doe'
+}
+template = sp.transmission.get('template_id', sub_data, True)
```

Comment:
I think we should create another method here called ``preview``. It can wrap/extend ``get`` if need be.